NosTests/Views/CompactNoteViewTests.swift

```diff
@@ -0,0 +1,29 @@
+import Dependencies
+import ViewInspector
+import XCTest
+
+final class CompactNoteViewTests: CoreDataTestCase {
+    @MainActor func testNewMediaDisplayDisabledUsesLinkPreviewCarousel() throws {
```

Comment:
Why no test for when the flag is enabled?

Reply:
Oh, I just realized why I didn't: I want to make sure everything continues to work as expected for the disabled case, since it should still be disabled. I don't really care what happens when it's enabled right now, since nothing is implemented for the new media display (and what _does_ happen is that we show an EmptyView, and testing for that seems unnecessary).

I did consider having an if/else block that just does the same thing in both cases (using the old media display) as a safety in case the new media display accidentally gets toggled on. And if I had done that, I'd be more interested in testing the enabled case.

Reply:
oh, that makes sense.